### DISCUSSION TITLE
Why does it takes screenshots even though I didn't write any screenshot code on browser mode?

### DISCUSSION THREAD

**Discussion by lesomnus** [vitest-dev/vitest#6228] [Q&A] [answered]
I have some tests files that runs on browser and some of tests takes screenshots.
How to disable screenshots?

**Comment by sheremet-va** [comment, marked as answer]
Vitest takes screenshots if tests fail. You can disable it with [`browser.screenshotFailures`](https://vitest.dev/config/#browser-screenshotfailures)

**Comment by ManAnRuck** [comment]
I have currently the same issue.
Documentation was changed and now I found the documentation here: https://vitest.dev/guide/browser/config.html#browser-screenshotfailures
:)
